csm/ImageSources.py

```diff
@@ -1,6 +1,6 @@
-import cv2
+# import cv2
```

Comment:
Let's try to keep the code in the repo meant to be run on the Jetson, can you uncomment these?

Reply:
Relic of past demo